.gitignore

```diff
@@ -0,0 +1,8 @@
+# Ignore everything
```

Comment:
Silly question - why do we want to ignore everything? If we are 'managing' the files through git, then I would have thought we'd like to be able to do a `git diff` and `git status` to see what has been modified, and thus needs committing - no?

Reply:
Well I have chosen this approach given the fact that our git repo on the Jenkins server is gonna be complicated to handle if we go the other way. If we ignore all the files on purpose, the .gitignore is gonna be more complex, but honestly I am not sure what's the best approach here.
@jodh-intel @sameo any thought ?